__tests__/integration.test.js

```diff
@@ -315,6 +315,24 @@ describe("/api/comments/:comment_id", () => {
     });
 });
 
+describe("/api/users", () => {
+    test("GET:200 Should send an array of all users", () => {
+        return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then((response) => {
+                const users = response.body.users;
+                expect(users.length).toBe(4);
+                users.forEach((user) => {
+                    expect(Object.keys(user).length).toBe(3);
```

Comment:
^^ what if later on you wanted to add more key-value pairs to users? I would suggest removing this specific expect and leaving the rest for flexibility 😄